x-pack/test/functional/apps/maps/sample_data.js

```diff
@@ -131,7 +131,7 @@ export default function ({ getPageObjects, getService, updateBaselines }) {
           'ecommerce_map',
           updateBaselines
         );
-        expect(percentDifference).to.be.lessThan(0.02);
```

Comment:
These are not timing differences but percent pixel difference in screen shots. The screen shots should not change at all from this PR so these values should not change.

Reply:
Actually, there is a difference, though.

<img width="409" alt="Screenshot 2022-03-21 at 20 11 43" src="https://user-images.githubusercontent.com/3940730/159337731-cbb23084-9472-46af-b5e6-9df63ba9aac0.png">
I think we should update the reference images.